tests/app/forms/field_handlers/test_address_handler.py

```diff
@@ -0,0 +1,80 @@
+from werkzeug.datastructures import MultiDict
+from wtforms import Form
+
+from app.forms import error_messages
+from app.forms.field_handlers import AddressHandler
+from app.forms.fields.address_field import AddressField
+from app.forms.validators import AddressLine1Required
+
+
+def get_test_form_class(answer_schema, messages=None):
+    handler = AddressHandler(answer_schema, error_messages=messages)
+
+    class TestForm(Form):
+        test_field = handler.get_field()
+
+    return TestForm
```

Comment:
```suggestion
    address_handler = AddressHandler(answer_schema, error_messages=messages)

    class TestForm(Form):
        test_field = address_handler.get_field()

    return TestForm
```

Reply:
either the above can change or the below can go from `address_handler` -> `handler`